src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/module/AccrualModule.java

```diff
@@ -17,4 +38,102 @@ protected AccrualModule(AccrualType accrualType) {
   public abstract BigDecimal calculateShiftContribution(ZonedDateTime startTime,
                                                         ZonedDateTime endTime);
 
+  public boolean applyTimeEntryToAccruals(TimeEntry timeEntry,
```

Comment:
I don't see why there would be a default implementation of this

Reply:
The `applyTimeEntryToAccruals` method is used by both night hours and annual target hours. This is why the decision was made to have a default implementation of this, within the AccrualModule class.